kentikapi/internal/api_payloads/device.go

```diff
@@ -336,6 +336,7 @@ func payloadToDeviceLabel(p deviceLabelPayload) (models.DeviceLabel, error) {
 	}, nil
 }
 
+// nolint: deadcode, unused
```

Comment:
@mateuszmidor Is it going to be used in the near future? Otherwise I'll remove the function

Reply:
You can remove it.

Reply:
Done